### PR TITLE
Want to talk copy changes

### PR DESCRIPTION
**Pull request by stevoland**

### CODE REVIEW

**Comment by asmega** [comment]
:+1:

**Comment by neoeno** [comment]
Is there a reason why there's no Welsh here?

**Comment by stevoland** [comment]
@neoeno It's only shown on `en` pages. Not sure why.

**Comment by stevoland** [comment]
@seane Agreed but what do I know.

File: app/views/shared/_want_to_talk.html.erb [hunk 1]
```diff
@@ -6,21 +6,21 @@
...
-      <%= t('want_to_talk.small_print') %>
-    </p>
+  </button>
+  <div class="want-to-talk__body" aria-hidden="true">
     <p class="want-to-talk__body__call-us">
       <%= t('want_to_talk.call_us') %>
     </p>
     <a href="<%= t('want_to_talk.phone_number') %>" class="want-to-talk__body__phone">
-      <%= t('want_to_talk.phone_number_readable') %>
+      <%= t('want_to_talk.phone_number_readable') %>*
```
Comment (benbarnett) [inline app/views/shared/_want_to_talk.html.erb]: Should this have the `phone_number` wrapped around in an anchor to make it clickable?

Comment (benbarnett) [inline app/views/shared/_want_to_talk.html.erb]: Oops - sorry ignore, I need to look up!

Comment (seane) [inline app/views/shared/_want_to_talk.html.erb]: For info, Service Delivery said there was no need for Welsh...which I
challenged, but their call!

**Pull request merged**